packages/mux-player/src/index.ts

```diff
@@ -145,6 +161,18 @@ class MuxPlayerElement extends VideoApiElement {
     //   // Temporarily here to load less segments on page load, remove later!!!!
     //   this.video.hls.config.maxMaxBufferLength = 2;
     // }
+
+    this.#setUpErrors();
+    this.#setUpCaptionsButton();
+    this.#setUpAirplayButton();
+    this.#setUpVolumeRange();
+    this.#monitorLiveWindow();
+    this.#userInactive = this.mediaController?.hasAttribute('user-inactive') ?? true;
+    this.#setUpCaptionsMovement();
```

Comment:
Non-blocking: This sort of ties into the conversation we had earlier. Constructors that do a lot of things generally feel like an anti-pattern in web component world. I'd be worried that any of these instance methods are calling `setAttribute` or otherwise making illegal DOM manipulations resulting in an error like: `Uncaught DOMException: Failed to construct 'CustomElement': The result must not have attributes`

Luckily, none of these methods are doing anything illegal, so that's good, but just wanted to call it out that it's a potential "gotcha".